client/blocks/importer/wordpress/upgrade-plan/hooks/use-get-upgrade-plan-site-metrics.tsx

```diff
@@ -1,39 +1,42 @@
 import { useIsEnglishLocale } from '@automattic/i18n-utils';
 import { getQueryArg } from '@wordpress/url';
 import { useUrlBasicMetricsQuery } from 'calypso/data/site-profiler/use-url-basic-metrics-query';
-import { upgradePlanSiteMetricsLcpThreshold } from '../constants';
+import {
+	upgradePlanSiteMetricsLcpThreshold,
+	upgradePlanSiteMetricsFidThreshold,
+} from '../constants';
+
+const calcPercentageDifferenceFromThreshold = ( value: number | undefined, threshold: number ) => {
+	if ( ! value || value <= threshold ) {
+		return 0;
+	}
+
+	return Math.round( 100 * Math.abs( ( value - threshold ) / ( ( value + threshold ) / 2 ) ) );
+};
 
 export const useGetUpgradePlanSiteMetrics = () => {
 	const isEnglishLocale = useIsEnglishLocale();
 	const importSiteQueryParam = getQueryArg( window.location.href, 'from' )?.toString() || '';
 	const { data: siteMetricData } = useUrlBasicMetricsQuery( importSiteQueryParam );
-	let showUpdatedSpeedMetrics = false;
+
 	let lcpPercentageDifference = 0;
+	let fidPercentageDifference = 0;
 
-	if (
-		isEnglishLocale &&
-		siteMetricData?.basic?.lcp &&
-		siteMetricData?.basic?.lcp > upgradePlanSiteMetricsLcpThreshold
-	) {
-		lcpPercentageDifference =
-			( siteMetricData?.basic?.lcp &&
-				Math.round(
-					100 *
-						Math.abs(
-							( siteMetricData?.basic?.lcp - upgradePlanSiteMetricsLcpThreshold ) /
-								( ( siteMetricData?.basic?.lcp + upgradePlanSiteMetricsLcpThreshold ) / 2 )
-						)
-				) ) ||
-			0;
+	if ( isEnglishLocale ) {
```

Comment:
I don't understand why, but I kept the english locale check for both (LCP and FID). Is it correct?

Reply:
This was mainly to allow us to push to production without waiting for the translations.

Reply:
@agrullon95 Quick question about translations if you don't mind. How will we know when translations are ready for all locales?

Reply:
You get a comment like this https://github.com/Automattic/wp-calypso/pull/78076#issuecomment-1604077351

Reply:
Thank you for the context and explanation, @agrullon95 and @alshakero! In this case, shouldn't we use the label to have the translations before the merge?

> Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?

From the PR checklist 👆

Reply:
Correct, you can set that label on the PR now to start the process.

Reply:
> You get a comment like this https://github.com/Automattic/wp-calypso/pull/78076#issuecomment-1604077351

We were expecting this comment in this PR https://github.com/Automattic/wp-calypso/pull/90966 and we didn't receive it.
I asked it here if you want to follow it: p1717434860923869-slack-C02AED43D